content/workshop/beginner-file.md

```diff
@@ -1,31 +1,28 @@
 ---
 title: "Listen to a folder"
-date: 2023-03-12T12:14:34+06:00
+date: 2024-01-24T12:14:34+06:00
 categories: ["beginner"]
 image: "/images/workshop/beginner-file/front.png"
 description: "Create a route that listens to a folder and outputs by log the modified files."
 draft: false
-kaoto: 1.0.0
+kaoto: 2.0.0
+vscode_kaoto: 0.11.0
 goals: learn how to listen to changes in a folder using Apache Camel
 ---
 
-## 1.- Log changes
+## 1 - Log changes
 
 The goals for this exercise are:
 
- - Start the flow with a `file-watch` step which watches a local folder like `/tmp/tutorial` and configure parameter `recursive` as `false`
+ - Create a new Camel route and let it start with a `file-watch` step, which will watch a local folder like `/tmp/tutorial/` and configure the parameter `recursive` as `false`, because we don't want to watch subfolders
  - Then `log` the detected change with an output like `Detected  ${header.CamelFileEventType} on file ${header.CamelFileName} at ${header.CamelFileLastModified}`
  
- **Make sure your folder (/tmp/tutorial) exists before running this integration.**
- 
- **This integration will work better when run locally, as the folder must be in the same machine that it gets deployed.**
-
 ### Hints
-
- - To add new steps to the canvas, take a look at the [User Guide](/docs/user-guide/).
- - To configure a step and fill the configuration properties, click on the step icon of the canvas.
- - The first step you want to add is called `file-watch`. Don't confuse it with `file`
- - The second step is a `log`. There may be more than one `log` option, choose the one without the camel logo for simplicity.
+Helpful advice for doing things better or more easily.
+- To add new, delete or replace steps on the canvas, right click an existing node. This will provide you a contextual menu.
+- To configure a step and fill the configuration properties, click on the step icon in the canvas.
+- The first step you want to add is called `file-watch`. Don't confuse it with `file`
+- The second step is a `log`. There may be more than one `log` option, choose the one without the camel logo for simplicity.
```

Comment:
The 2 logs have now the same logo. We should precise if we suggest to use a Component or a Processor

Reply:
good point

Reply:
fixed